canonical-web-and-design/ubuntu.com
# Build blender support purchase modal

## Done
Built the blender support purchase modal

## QA
- Go to https://ubuntu-com-10711.demos.haus/advantage/subscribe/blender?test_backend=true
- Fill out the form and make a purchase
- It should work
- Go to https://ubuntu-com-10711.demos.haus/advantage/subscribe?test_backend=true
- Fill out the form and make a purchase
- It should work

## Issue
Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/315, https://github.com/canonical-web-and-design/commercial-squad/issues/350
